source/partials/release_notes/_release-20-6-0.md.erb

```diff
@@ -0,0 +1,65 @@
+<h4>Add Pipeline-as-code creation page</h4>
+
+Now the GoCD users can create config-repo pipeline definitions using the new SPA.
+
+<h4>Removed support for SSL </h4>
+
+Starting GoCD 20.6.0, complete support for SSL has been removed. This is in accordance with the announcement made in GoCD 20.2.0.
+
+Please read the [GoCD SSL Changes](https://github.com/gocd/gocd/issues/7872) for more information.
```

Comment:
cc:\ @maheshp

Reply:
<h4>SSL support - Breaking Changes</h4>

In GoCD 20.2.0 we had [announced](https://github.com/gocd/gocd/issues/7872) removal of support for TLS on port 8154 using a self-signed certificate. To minimize disruptions, GoCD temporarily provided an option to enable the TLS port.
With this release, GOCD no longer support configuring TLS within GoCD and will stop bringing up TLS on port 8154. If you have temporarily enabled TLS, and if the agents are configured to use the TLS port 8154 they will have to reconfigured to use port 8153, else the agent will stop connecting to the server.

Reply:
Thanks